bindings/wasm/test/cdn.test.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright 2025 The Manifold Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {resolve} from 'node:path';
+import {beforeEach, expect, suite, test} from 'vitest';
+
+import {bundleFile} from '../lib/bundle.ts';
+import * as worker from '../lib/worker.ts';
+
+let evaluator = worker.getEvaluator() || worker.initialize();
+beforeEach(() => worker.cleanup());
+
+suite('Import remote modules from', () => {
+  test('esm.sh', async () => {
+    const entrypoint =
+        resolve(import.meta.dirname, './fixtures/importFromCDN.mjs');
+    const bundle = await bundleFile(entrypoint, {jsCDN: 'esm.sh'});
+    const result = await evaluator.evaluate(bundle);
+    expect(result.volume()).toBeGreaterThan(0);
```

Comment:
Nice end-to-end testing! I'm slightly concerned about our CI being dependent on external websites, but then again, I think the CDNs have much more stringent uptime requirements than we do, so I think this is good.

Reply:
Honestly, I'd rather err on the side of more test coverage in the CI than less. If we start seeing flakes, let's reevaluate then.